src/pages/CampusAlumniEndowmentStudioPage.tsx

```
@@ -0,0 +1,218 @@
+                  <label className="block text-xs font-bold text-slate-300 mb-1">Fund Name</label>
+                  <input type="text" required value={newFundName} onChange={(e) => setNewFundName(e.target.value)} className="w-full px-3.5 py-2 rounded-xl bg-slate-950 border border-slate-800 text-white text-sm" />
+                </div>
+
+                <div className="grid grid-cols-2 gap-3">
+                  <div>
+                    <label className="block text-xs font-bold text-slate-300 mb-1">Donor Name</label>
+                    <input type="text" required value={newDonor} onChange={(e) => setNewDonor(e.target.value)} className="w-full px-3.5 py-2 rounded-xl bg-slate-950 border border-slate-800 text-white text-sm" />
+                  </div>
+                  <div>
+                    <label className="block text-xs font-bold text-slate-300 mb-1">Graduation Year</label>
+                    <input type="number" required value={newYear} onChange={(e) => setNewYear(parseInt(e.target.value))} className="w-full px-3.5 py-2 rounded-xl bg-slate-950 border border-slate-800 text-white text-sm" />
```

ESLint

[error] Replace `·type="number"·required·value={newYear}·onChange={(e)·=>·setNewYear(parseInt(e.target.value))}·className="w-full·px-3.5·py-2·rounded-xl·bg-slate-950·border·border-slate-800·text-white·text-sm"` with `⏎······················type="number"⏎······················required⏎······················value={newYear}⏎······················onChange={(e)·=>·setNewYear(parseInt(e.target.value))}⏎······················className="w-full·px-3.5·py-2·rounded-xl·bg-slate-950·border·border-slate-800·text-white·text-sm"⏎···················`

(prettier/prettier)